packages/sdk/identity/README.md

```diff
@@ -2,4 +2,4 @@
 
 @celo/identity simplifies interacting with ODIS, Celo’s lightweight identity layer based on phone numbers.
 
-You can find an example of how to use this package to do ODIS lookups [here](https://github.com/critesjosh/register-number).
+You can find an example of how to use this package to do ODIS lookups [here](https://github.com/celo-org/emisianto/blob/main/pages/index.tsx).
```

Comment:
did you mean to just link to a TSX file?

Reply:
yeah because the rest of the code is specific to Next.js, the `index.tsx` files linked here gives the person looking at the example a good idea how the `@celo/identity` packages is supposed to be used.